src/utils/userHelpers.ts

```diff
@@ -0,0 +1,3 @@
+export const isUserAdmin = (user: { role: number }) => {
```

Comment:
it might be worth considering putting this somewhere in src/lib, e.g. a src/lib/users.ts file, as a start to gathering the "model logic" in one place rather than scattering it across the frontend app.

Reply:
i'm going to do this now on the org-view branch, actually -- feel free to leave this as-is and we can clean up later

Reply:
Done.